### DISCUSSION TITLE
errors with HNN GUI installation

### DISCUSSION THREAD

**Discussion by stephanie-r-jones** [jonescompneurolab/hnn#325] [Q&A] [answered]
HNN GUI was working on my mac after pip installation but II haven't run it in a while and now ran into the following  into errors: 
****   
 Fatal Python error: initfsencoding: "unable to load the file system codec
ModuleNotFoundError: No module named 'encodings'
Current thread 0x000000010807f5c0 (most recent call first):
Abort trap: 6
****

I then went back and ran a pre-install check and received the following warnings: 
**** 
Checking PYTHONPATH environment variable is not set...warning
NEURON libraries found in PYTHONPATH variable. This was used before 
the 'pip install' method. Make sure this matches the version used to 
compile HNN modules with 'make'
PYTHONPATH=/Applications/NEURON-7.8/lib/python:

Checking PYTHONHOME environment variable is not set...warning

PYTHONHOME is set. This is not the recommended way to set the python environment.
Make sure that this is the correct environment to run HNN.
PYTHONHOME=/Applications/anaconda
Fatal Python error: initfsencoding: unable to load the file system codec
ModuleNotFoundError: No module named 'encodings'

Current thread 0x00000001079125c0 (most recent call first):
Python version: 

Checking NRN_PYLIB environment variable is not set...ok
Checking for existing hnn directory...found
Existing HNN source code directory found at /Users/stephaniejones/hnn_new
Optional: use this directory instead of step "Download HNN source code" 
****

**Comment by jasmainak** [comment]
can you say what are the following variables:

```sh
$ echo PYTHONHOME
$ echo PYTHONPATH
$ which python
```

**Comment by stephanie-r-jones** [comment]
> On Jul 26, 2022, at 10:33 AM, Mainak Jas ***@***.***> wrote:
> 
> 
> can you say what are the following variables:
> 
> $ echo PYTHONHOME
PYTHONHOME
> $ echo PYTHONPATH
PYTHONPATH
> $ which python

/Applications/anaconda/envs/hnn/bin/python

**Comment by jasmainak** [comment, marked as answer]
The solution was to remove all references to PYTHONHOME and PYTHONPATH in bashrc and bash_profile

**Reply by rythorpe** [reply]
Or if your mac uses zsh instead of bash like the latest OS is defaulting to these days, ~/.zshrc and ~/.zprofile.
